frontend/src/metabase/dashboard/components/DashboardHeader/tests/setup.tsx

```diff
@@ -56,8 +56,11 @@ export const setup = async ({
     "token-features": createMockTokenFeatures(tokenFeatures),
   });
 
-  if (hasEnterprisePlugins) {
-    setupEnterprisePlugins();
+  if (hasEnterprisePlugins && tokenFeatures?.audit_app) {
+    setupEnterpriseOnlyPlugin("audit_app");
```

Comment:
This looks in contrast with other tests that accept `specificPlugins` from the test, not in the setup function. Should we pass `specificPlugins` instead?